settings/DevHome.Settings/Views/AccountsPage.xaml.cs

```diff
@@ -208,4 +210,25 @@ private async void Logout_Click(object sender, RoutedEventArgs e)
             _ = await afterLogoutContentDialog.ShowAsync();
         }
     }
+
+    private async Task InitiateAddAccountUserExperienceAsync(Page parentPage, AccountsProviderViewModel accountProvider)
+    {
+        if (accountProvider.DeveloperIdProvider.GetAuthenticationExperienceKind() == Microsoft.Windows.DevHome.SDK.AuthenticationExperienceKind.CardSession)

```

Comment:
You probably want to store `accountProvider.DeveloperIdProvider.GetAuthenticationExperienceKind()` in a variable rather than calling it for each if clause.

Reply:
Addressed in latest commit. Thanks!